pkg/api/api.go

```diff
@@ -260,6 +260,7 @@ func (a *API) RegisterRuntimeConfig(runtimeConfigHandler http.HandlerFunc) {
 func (a *API) RegisterDistributor(d *distributor.Distributor, pushConfig distributor.Config) {
 	distributorpb.RegisterDistributorServer(a.server.GRPC, d)
 
+	a.RegisterRoute("/api/v1/push/otlp", push.OTLPHandler(a.sourceIPs, a.cfg.wrapDistributorPush(d)), true, "POST")
```

Comment:
https://github.com/prometheus/prometheus/blob/main/web/api/v1/api.go#L388
Can we align with Prometheus OTLP metrics endpoint?

Reply:
did /api/v1/otlp/v1/metrics like the docs say https://prometheus.io/docs/prometheus/latest/querying/api/#otlp-receiver. I hope that is ok